pages/badges/badges.jsx

```diff
@@ -155,6 +155,7 @@ var BadgesPage = React.createClass({
   },
 
   generateBadgeList: function() {
+    var that = this;
```

Comment:
yeah, at this point it means that map function should be refactor out of the current code into its own component function instead.